### ISSUE TITLE
UFO sources in TrueType

### ISSUE THREAD

**Issue by tphinney** [tphinney/science-gothic#189]
Google would like the UFO source files to be in TrueType

Unfortunately, doing that automatically at export-to-UFO-time is not yielding good results with FontMake to variable TTF workflow.

Vassil, can you investigate this problem next week?

**Comment by kateliev** [comment]
Seeing this little late, but can you define:
> not yielding good results

- outlines broken
- compatibility broken
- bad conversion of curves
....

...in the meantime will try it out and report next week!

**Comment by tphinney** [comment]
Compatibility broken, is what I recall. If I exported UFO with TT outlines from FontLab, *many* of those glyphs failing to build in variable TTF with FontMake.

**Comment by kateliev** [comment]
Tested several options - the best way to approach this is as follows:
PS to TT conversion at export time will not suit our needs - it looks like FL does not care about conversion being "compatible" between exported UFO files. FontMake returns a lot of compatibility errors. The solution is to convert the font into TT with Actions > Convert Curves to TT. Then test for compatibility using the "Font QA Test" script. It will list few characters that have to be fixed by hand. After fixing compatibility, just export as usuall and it works.

The bottom line - in font TT conversion is key!

**Comment by tphinney** [comment]
OK. I will have to see how time-consuming this is. It may be that we will have to permanently convert our FontLab source files to TT outlines along the way.  :/

**Comment by tphinney** [comment]
Well, _that_ was unexpected: whether or not I check the pref for TT Conversion to "Add nodes every two TT off-curve points", the resulting file passes the Font QA script with NO errors! 🕺

**Comment by tphinney** [comment]
So, this is good and done.

Just need to start checking in the UFO intermediates as well as the final TTFs. I will close this when I add this to the routine.

**Comment by tphinney** [comment]
I just need to remember to check in the UFO sources. Could do that already for the last variable TTF generated.

**Comment by tphinney** [comment]
Done for last build. Will continue for future builds.
